Security-Team.md

```diff
@@ -60,7 +60,7 @@ These non-TSC and TSC Emeriti also have access:
 * [vdeturckheim](https://github.com/vdeturckheim) - **Vladimir de Turckheim**
 * [@watson](https://github.com/watson) - **Thomas Watson**
 
-List is from the [member page](https://hackerone.com/nodejs/team\_members) for
+List is from the [member page](https://hackerone.com/nodejs/team_members) for
```

Comment:
functionally works, but I still find this escape an odd one. I realize it's coming from the auto-fix though

Reply:
The escape is being removed, not added. That's because we updated dependencies, and thanks to https://github.com/syntax-tree/mdast-util-to-markdown/commit/7b381da5bceee3c36be079e27a2bad7b3cc412c2, the unnecessary escapes no longer happen in these situations.

Reply:
Right, miss-read that